app/packages/state/src/recoil/schema.ts

```
@@ -320,6 +322,66 @@ export const dbPath = selectorFamily({
+
+      // todo: frames
+      if (space) {
+        return space === State.SPACE.FRAME
+          ? getLabelFields(get(atoms.frameFields), "frames.")
+          : getLabelFields(get(atoms.sampleFields));
+      }
+
+      return [
+        ...getLabelFields(get(atoms.sampleFields)),
+        ...getLabelFields(get(atoms.frameFields), "frames."),
+      ];
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement, ...

... this statement, ...

... this statement, ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)